JS/Lesson31/saffet/my-app/src/components/CreateTask/index.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+import './style.css';
+
+export const CreateTask = ({ onCreateProps }) => {
+
+  const [task, setTask] = useState("");
+
+  const handleInputChange = (event) => {
+    setTask(event.target.value);
+  }
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log(task);
+
```

Comment:
I'd modify this function a bit so we only add a new todo item if there's any text:
```
if (task.length > 0) {
      onCreateProps(task);
      setTask("");
    }
```